docs/source/includes/elasticluster_setup.rst

```diff
@@ -0,0 +1,352 @@
+
+.. code:: ini
+
+   # Gridengine software to be configured by Ansible
+   [setup/ansible-gridengine]
+   provider=ansible
+   frontend_groups=gridengine_master
+   compute_groups=gridengine_clients
+   
+   # Create a cloud provider (call it "google-cloud")
+   [cloud/google-cloud]
+   provider=google
+   gce_project_id=****REPLACE WITH YOUR PROJECT ID****
+   gce_client_id=****REPLACE WITH YOUR CLIENT ID****
+   gce_client_secret=****REPLACE WITH YOUR SECRET KEY****
+   
+   # Create a login (call it "google-login")
+   [login/google-login]
+   image_user=****REPLACE WITH YOUR GOOGLE USERID (just the userid, not email)****
+   image_user_sudo=root
+   image_sudo=True
+   user_key_name=elasticluster
+   user_key_private=~/.ssh/google_compute_engine
+   user_key_public=~/.ssh/google_compute_engine.pub
+   
+   # Bring all of the elements together to define a cluster called "gridengine"
+   [cluster/gridengine]
+   cloud=google-cloud
+   login=google-login
+   setup_provider=ansible-gridengine
+   security_group=default
+   image_id=****REPLACE WITH OUTPUT FROM: gcloud compute images list | grep ^backports-debian | cut -f 1 -d " "****
+   flavor=n1-standard-1
+   frontend_nodes=1
+   compute_nodes=3
+   image_userdata=
+   ssh_to=frontend
+
+Setting the boot disk size
+**************************
+For cluster tasks you may want to create use SSD persistent disk or a boot disk larger than the default 10 GB.
+Elasticluster allows for specifying both the boot disk type and size for instances of your cluster:
+
+``boot_disk_type``
+    Define the type of boot disk to use.
+    Supported values are `pd-standard` and `pd-ssd`.
+    Default value is `pd-standard`.
+
+``boot_disk_size``
+    Define the size of boot disk to use.
+    Values are specified in gigabytes.
+    Default value is 10.
+
+The disk type and size can be set for a cluster or for a group of nodes.
+
+For example to set up the above Grid Engine cluster such that nodes have a 100 GB
+SSD `persistent disk`_, add the following:
+
+.. code:: ini
+
+   [cluster/gridengine]
+   ...
+   boot_disk_type=pd-ssd
+   boot_disk_size=100
+
+or to configure all of the ``compute`` worker nodes to have a 2 TB Standard (HDD) `persistent disk`_:
+
+.. code:: ini
+
+   [cluster/gridengine/compute]
+   boot_disk_type=pd-standard
+   boot_disk_size=2000
+
+Generating your SSH keypair
+***************************
+
+If you have already connected to a Google Compute Engine instance using ``gcloud ssh``, then you will have a keypair generated in:
+
+* ``~/.ssh/google_compute_engine``
+* ``~/.ssh/google_compute_engine.pub``
+
+If you do not have a keypair, then the ``user_key_private`` and ``user_key_public`` file paths in the ``[login/google-login]`` section above will not be valid.
+
+You can create a test instance and connect to it with ``gcloud ssh``, or generate your keypair manually with:
+
+.. code:: bash
+
+   ssh-keygen -t rsa -f ~/.ssh/google_compute_engine
```

Comment:
Interesting. I was not interested in using it because it does more:

    gcloud compute config-ssh makes SSHing to virtual machine instances easier
    by adding an alias for each instance to the user SSH configuration
    (~/.ssh/config) file.

but if you have no instances, then that part is a no-op.

Thanks!